### PR TITLE
Ensure `nvcc` version matches `cudatoolkit` version

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR ensures that the `nvcc` version installed from `conda` matches the `cudatoolkit` version installed from `conda`.

### CODE REVIEW

**Review by sevagh — APPROVED** [review]

File: rapidsai/devel-centos.Dockerfile
Comment (jakirkham) [inline rapidsai/devel-centos.Dockerfile:91]: Wonder if we should also add a comment to the source code explaining this as well

Comment (ajschmidt8) [reply rapidsai/devel-centos.Dockerfile:91]: seems reasonable. I don't want to interrupt the current CI run, so I'll let that run through and add the comment after followed by a force merge.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks AJ! 🙏

**Pull request merged**
